src/aks-preview/HISTORY.md

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+0.0.1
```

Comment:
should be 0.1.0 now to be ready for release

Reply:
yes, sharp eyes.